test/python/api_config.py

```diff
@@ -53,6 +53,7 @@ def setUpClass(cls):
         cls.account = os.environ[CONJUR_ACCOUNT]
 
         cls.client = get_api_client()
+        cls.bad_client = openapi_client.ApiClient(get_api_config())
```

Comment:
We should add some comments here to describe what makes one "bad" and the other "good"